### DISCUSSION TITLE
Help scaling dff model.

### DISCUSSION THREAD

**Discussion by gsimiya** [Parik27/DragonFF#309] [Q&A] [answered]
I just downloaded a [skin mod](https://www.gtainside.com/en/sanandreas/skins/184601-doaxvv-honoka-ryza-s-favorite-outfit/), but the size is too big. So I wanted to make it smaller.

![2024-06-09 14_36_34-GTA_ San Andreas _ 1920x1080 @ 61 fps](https://github.com/Parik27/DragonFF/assets/172172025/bb9bc308-fefb-4046-bbaa-f85c8de3434e)
as you can see, the model is too tall and the head is too big.

What I tried is, import the dff file, scale the object, then export the dff file and use it to replace the original file.
But what happened is, not only the model is not downscaled, the texture was not loaded.

![2024-06-09 14_39_10-GTA_ San Andreas _ 1920x1080 @ 61 fps](https://github.com/Parik27/DragonFF/assets/172172025/2c219220-5bfc-445c-9700-d47d65d699dd)

Can anyone guide me on how to downscale the model?

**Comment by Parik27** [comment, marked as answer]
Your model was likely made in KAMS, so use Use Material Split option while importing. Also you need to apply transformations after scaling or edit in edit mode.

**Comment by gsimiya** [comment]
that worked!! thank you very much.
this will be fun
